### PR TITLE
Add ability to process Wikibot logs

### PR DESCRIPTION
**Pull request by anders-sandholm**
Code added for processing log record files from running the Wikibot.

### CODE REVIEW

File: python/wikibot/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+precision_map = {
+  sling.MILLENNIUM: pywikibot.WbTime.PRECISION['millenia'],
+  sling.CENTURY: pywikibot.WbTime.PRECISION['century'],
+  sling.DECADE: pywikibot.WbTime.PRECISION['decade'],
+  sling.YEAR: pywikibot.WbTime.PRECISION['year'],
+  sling.MONTH: pywikibot.WbTime.PRECISION['month'],
+  sling.DAY: pywikibot.WbTime.PRECISION['day']
+}
+
+class Dashboard:
```
Comment (ringgaard) [inline python/wikibot/dashboard.py]: We usually use "dashboard" about something with a user interface. What about renaming this to something like WikiMonitor?

File: python/wikibot/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    test = ""
+    if flags.arg.test: test = "test-"
+    self.pattern = self.path + "wikibotlog-" + test + "20*.rec"
+
+    self.store = sling.Store()
+    self.n_item = self.store["item"]
+    self.n_facts = self.store["facts"]
+    self.n_status = self.store["status"]
+    self.n_revision = self.store["revision"]
+    #self.n_url = self.store["url"]
```
Comment (ringgaard) [inline python/wikibot/dashboard.py]: pls delete this test line

File: python/wikibot/dashboard.py (hunk 1 above)
Comment (anders-sandholm) [reply python/wikibot/dashboard.py]: Agree. Done

File: python/wikibot/dashboard.py (hunk 2 above)
Comment (anders-sandholm) [reply python/wikibot/dashboard.py]: done

**Pull request merged**